src/compiler/parse/state/tag.ts

```diff
@@ -160,8 +165,9 @@ export default function tag(parser: Parser) {
 		};
 	}
 
+	
 	const unique_names: Set<string> = new Set();
-
+	
```

Comment:
no need to add whitespace on this line. I also probably remove lots of the new lines being added in this PR
```suggestion

```